lib/vrl/cli/src/cmd.rs

```diff
@@ -191,6 +191,7 @@ fn repl(objects: Vec<Value>, timezone: TimeZone, vrl_runtime: VrlRuntime) -> Res
 }
 
 #[cfg(not(feature = "repl"))]
+#[allow(clippy::needless_pass_by_value)]
```

Comment:
It's probably self-explanatory enough here, but generally it is nice to see comments for these allows to make it clear why we are allowing.